verhoef2021reproducibility/journal/metadata.yaml

```diff
@@ -5,57 +5,46 @@
 #  - For a successful replication, it should be prefixed with "[Re]"
 #  - For a failed replication, it should be prefixed with "[¬Re]"
 #  - For other article types, no instruction (but please, not too long)
-title: "ReScience (R)evolution"
+title: "[Re] Reproducibility study - Does enforcing diversity in hidden states of LSTM-Attention models improve transparency?"
 
 # List of authors with name, orcid number, email and affiliation
 # Affiliation "*" means contact author (required even for single-authored papers)
 authors:
-  - name: Konrad Hinsen
-    orcid: 0000-0003-0330-9428
-    email: konrad.hinsen@cnrs.fr
-    affiliations: 1,2
-    
-  - name: Nicolas P. Rougier
-    orcid: 0000-0002-6972-589X
-    email: Nicolas.Rougier@inria.fr
-    affiliations: 3,4,5,*      # * is for contact author
+  - name: Pieter Bouwman
+    orcid: 0000-0002-1697-792X
+    email: pieter.bouwman@student.uva.nl
+    affiliations: 1,*      # * is for contact author
+  - name: Yun Li
+    orcid: 0000-0002-2715-3168
+    email: liyun.elf@gmail.com
+    affiliations: 1
+  - name: Rogier van der Weerd
+    orcid: 0000-0002-7791-2580
+    email: rogier.van.der.weerd@student.uva.nl
+    affiliations: 1
+  - name: Frank Verhoef
+    orcid: 0000-0001-6235-7177
+    email: fsa.verhoef@chello.nl
+    affiliations: 1
 
 # List of affiliations with code (corresponding to author affiliations), name
 # and address. You can also use these affiliations to add text such as "Equal
 # contributions" as name (with no address).
 affiliations:
   - code:    1
-    name:    Centre de Biophysique Moléculaire, CNRS UPR4301
-    address: Orléans, France
-    
-  - code:    2
-    name:    Synchrotron SOLEIL, Division Expériences
-    address: Gif sur Yvette, France
-    
-  - code:    3
-    name:    INRIA Bordeaux Sud-Ouest
-    address: Bordeaux, France
-    
-  - code:    4
-    name:    LaBRI, Université de Bordeaux, Institut Polytechnique de Bordeaux,
-             Centre National de la Recherche Scientifique, UMR 5800
-    address: Talence, France
-    
-  - code:    5
-    name:    Institut des Maladies Neurodégénératives, Université  de Bordeaux,
-             Centre National de la Recherche Scientifique, UMR 5293
-    address: Bordeaux, France
+    name:    University of Amsterdam
+    address: Amsterdam, The Netherlands
 
 
 # List of keywords (adding the programming language might be a good idea)
-keywords: rescience c, rescience x
+keywords: Attention, NLP, Transparency, Explainability, Faithfulness, Plausibility, Reproducibility, LSTM, ReScience
 
 # Code URL and DOI/SWH (url is mandatory for replication, doi after acceptance)
 # You can get a DOI for your code from Zenodo, or an SWH identifier from
 # Software Heritage.
 #   see https://guides.github.com/activities/citable-code/
 code:
-  - url: https://github.com/rescience-c/template
+  - url: https://anonymous.4open.science/r/FACT_AI_project/
   - doi: 
   - swh: 
```

Comment:
1. Missing SWH identifier
2. Since the paper is now accepted, you may change the URL to a non-anonymous link if you like!